src/labels/CustomLabels.labels

```diff
@@ -4880,6 +4880,98 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Error completing the specified API call</shortDescription>
         <value>Error completing the {0} api: {1}</value>
     </labels>
+    <labels>
+        <fullName>batchNumberSettingsDescActivation</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of how to use the existing batch number format table</shortDescription>
+        <value>Activate or deactivate batch number formats. One batch number format can be active at time.</value>
+    </labels>
+    <labels>
+        <fullName>batchNumberSettingsHeaderFormats</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header for table of existing batch number formats</shortDescription>
+        <value>Batch Number Formats</value>
+    </labels>
+    <labels>
+        <fullName>batchNumberSettingsSave</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link to save a batch number format</shortDescription>
+        <value>Save Batch Number Format</value>
+    </labels>
+    <labels>
+        <fullName>batchNumberSettingsDescDisplayFormat</fullName>
+        <categories>Batch Number Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Explanation of Display Format options</shortDescription>
+        <value>Display Format allows you to control the appearance of your auto-numbered field. It must be unique and include the number of digits as well as a prefix, if desired. Your format can be up to 30 characters.
+            &#10;Examples
+            *
```

Comment:
```suggestion
            <ul>
                 <li>A sequence formatted as {0} with a starting number of 9 displays as 9, 10, 11, etc.</li>
```

Reply:
@kenlewis-sfdo I'm not sure I did these unordered list item suggestions correctly. How's this look?